editor/editor.gd

```diff
@@ -269,8 +269,15 @@ func _get_file_dialog() -> ConfirmationDialog:
 
 
 func _command_button_list_pressed(command_script:Script) -> void:
+	var command_idx:int = -1
+	var tree_item:TreeItem = timeline_displayer.get_selected()
+	if tree_item:
+		var selected:Command = timeline_displayer.get_selected().get_metadata(0)
+		if selected:
+			command_idx = _current_timeline.get_command_idx(selected) + 1
```

Comment:
TimelineDisplayer now uses CommandCollection, which uses `get_command_index` and `get_command_absolute_position`